package.json

```diff
@@ -8,7 +8,8 @@
     "build": "razzle build",
     "test": "razzle test --env=jsdom",
     "start": "NODE_ENV=production node build/server.js",
-    "speculate": "speculate"
+    "speculate": "speculate",
+    "cypress": "cypress open"
```

Comment:
Does this run the tests? It should be more specific to what it is doing, something like `test:e2e`.

Reply:
The default should just be running a command and the tests automatically run (CI will use that too), there shouldn't need to be an extra step. We should add both, and give the ability to be more interactive.